### DISCUSSION TITLE
copilot on VS Code

### DISCUSSION THREAD

**Discussion by siofi69** [community/community#142362] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Facing difficulties with Copilot on VS Code, anyone got any suggestions ?

**Comment by master-coco** [comment, marked as answer]
go to extensions on VS code and see if you have copilot installed or enabled 
it should show up at the bottom right side of your IDE
I have the github student program thing so we get copilot for free but you should get it once you login to your account

**Reply by siofi69** [reply]
Thank you  for the prompt response
